nvidia.go

```diff
@@ -29,6 +29,11 @@ func getDevices() []*pluginapi.Device {
 		devs = append(devs, &pluginapi.Device{
 			ID:     d.UUID,
 			Health: pluginapi.Healthy,
+			Topology: &pluginapi.TopologyInfo{
+				Node: &pluginapi.NUMANode{
```

Comment:
Thanks for this PR. Would you mind updating this to the newest API interface?

https://github.com/kubernetes/kubernetes/commit/1c1f19c61c18b7176674dce3ff9f64248fe74a89#diff-45056b94548d2ceef7f0d1028f937a05L77

Reply:
done.